api/handler/delete.go

```diff
@@ -17,6 +17,8 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+const maxObjectToDelete = 1000
```

Comment:
Issue number in the comment will be appreciated for the specification details.

Reply:
Maybe we should add comment with link to the aws doc?

Reply:
Yes, but link can be unavailable one day, and PR will remain with the link anyway. I am good either way.